index.js

```diff
@@ -72,6 +72,18 @@ class GhostInspector {
     }
   }
 
+  getOverallScreenshotOutcome(data) {
+    if (data instanceof Array) {
+      let passing = data.length ? true : null
+      for (const entry of data) {
+        passing = passing && entry.screenshotComparePassing
+      }
+      return passing
+    } else {
+      return data.screenshotComparePassing === undefined ? null : data.screenshotComparePassing
+    }
+  }
+
```

Comment:
Maybe better to just reuse `getOverallResultOutcome()` and do `getOverallResultOutcome(data, field = 'passing')` by default, then override with `getOverallResultOutcome(data, 'screenshotComparePassing')`? Otherwise it's the exact same logic.

Reply:
Yeah, I had them split out originally because of the dependent logic, but maybe it makes sense to put them in the same function. 🤔  I'll switch it over...